Budget sweep is broken

```
Traceback (most recent call last):
  File "experiments/experiment_budget_sweep.py", line 338, in <module>
    result_dict[SolveStrategy.APPROX_RANDOMIZED_ROUND] = get_futures(futures, desc="LP approx rand")
  File "/data/paras/checkmate/experiments/common/ray_utils.py", line 11, in get_futures
    results.extend(ray.get(done_results))
  File "/data/paras/miniconda3/lib/python3.7/site-packages/ray/worker.py", line 2349, in get
    raise value
ray.exceptions.RayTaskError: ray_worker:remat.core.solvers.strategy_approx_lp.solve_approx_lp_randomized() (pid=33236, host=c77)
  File "/data/paras/checkmate/remat/core/solvers/strategy_approx_lp.py", line 154, in solve_approx_lp_randomized
    write_model_file=write_model_file,
  File "/data/paras/checkmate/remat/core/solvers/strategy_optimal_ilp.py", line 38, in __init__
    assert not self.solve_r, "Can't solve for R if producing a fractional solution"
AssertionError: Can't solve for R if producing a fractional solution
```

--- a/remat/core/solvers/strategy_approx_lp.py
+++ b/remat/core/solvers/strategy_approx_lp.py
@@ -48,7 +48,6 @@ def solve_approx_lp_deterministic_sweep(
         eps_noise=eps_noise,
         imposed_schedule=imposed_schedule,
         write_model_file=write_model_file,
-        solve_r=False,
     )
     lpsolver.build_model()
     try:
@@ -150,10 +149,10 @@ def solve_approx_lp_randomized(
         int(0.9 * budget),  # hack to get values under the budget
         gurobi_params=param_dict,
         seed_s=seed_s,
+        solve_r=False,
         integral=False,
         eps_noise=eps_noise,
         write_model_file=write_model_file,
-        solve_r=False,
     )
     lpsolver.build_model()
     try:

